src/ZfcUser/Factory/Form/ChangeEmail.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Clayton Daley
+ * Date: 3/10/2015
+ * Time: 9:34 AM
+ */
+
+namespace ZfcUser\Factory\Form;
+
+use Zend\Form\FormElementManager;
+use Zend\ServiceManager\FactoryInterface;
+use Zend\ServiceManager\ServiceLocatorInterface;
+use ZfcUser\Form;
+use ZfcUser\Validator;
+
+class ChangeEmail implements FactoryInterface
+{
+    public function createService(ServiceLocatorInterface $formElementManager)
+    {
+        if ($formElementManager instanceof FormElementManager) {
```

Comment:
If the element is registered in formElementManager it's always the formElementManager you'll get injected, so i'm not sure i see the use for this `if` check?